### PR TITLE
LPD-17637-17639-17640 Fixed functional tests by swapping them for Playwrigh ones

### PR DESCRIPTION
**Pull request by albertojml**
**What's new?:**
- Added new Playwright tests that substitute the previously failing Poshi ones.

**What's changed?:**
- Deleted the failing Poshi tests.

See the following **Jira tickets**: [LPD-17637](https://liferay.atlassian.net/browse/LPD-17637), [LPD-17639](https://liferay.atlassian.net/browse/LPD-17639), [LPD-17640](https://liferay.atlassian.net/browse/LPD-17640).

**NOTE:** The first commit of this PR is https://github.com/liferay-headless/liferay-portal/commit/a2db56d72b851871a7b9230a5e66fffb296b653d

### CODE REVIEW

**Comment by albertojml** [comment]
ci:test:playwright

File: modules/test/playwright/tests/headless-builder-web/get.spec.ts [hunk 1]
```diff
@@ -95,6 +95,69 @@ test('can see path parameter property with map details', async ({
...
+
+	await page.getByRole('button', {name: 'Select an Option'}).click();
+	await expect(page.getByRole('menuitem', {name: 'Base URL'})).toBeVisible();
+	await expect(
+		page.getByRole('menuitem', {name: 'External Reference Code'})
+	).toBeVisible();
+	await expect(page.getByRole('menuitem', {name: 'ID'})).toBeVisible();
+	await expect(page.getByRole('menuitem', {name: 'Title'})).toBeVisible();
+
+	await page.goto('/');
```
Comment (4lejandrito) [inline modules/test/playwright/tests/headless-builder-web/get.spec.ts:154]: Not needed.

File: modules/test/playwright/tests/headless-builder-web/apiExplorer.spec.ts [hunk 2]
```diff
@@ -71,7 +86,7 @@ test('can see filter and sort parameters for collection endpoints', async ({
 	);
 });
 
-test('cannot see filter and sort parameters for singleElement endpoints', async ({
+test('can see get endpoint path with erc parameter in api explorer', async ({
```
Comment (4lejandrito) [inline modules/test/playwright/tests/headless-builder-web/apiExplorer.spec.ts:89]: `in api explorer` is redundant.

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

**Comment by albertojml** [comment]
ci:test:sf

**Comment by albertojml** [comment]
ci:test:playwright

**Comment by albertojml** [comment]
ci:forward:force

**Pull request closed without merging**